### PR TITLE
Trust return value nullability again for gsk

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

File: gsk4/src/auto/transform.rs [hunk 1]
```diff
@@ -60,7 +60,7 @@ impl Transform {
 
     #[doc(alias = "gsk_transform_perspective")]
     #[must_use]
-    pub fn perspective(&self, depth: f32) -> Option<Transform> {
+    pub fn perspective(&self, depth: f32) -> Transform {
```
Comment (sdroege) [inline gsk4/src/auto/transform.rs:63]: This is correct and can't return NULL

File: gsk4/src/auto/transform.rs [hunk 2]
```diff
@@ -49,7 +49,7 @@ impl Transform {
 
     #[doc(alias = "gsk_transform_matrix")]
     #[must_use]
-    pub fn matrix(&self, matrix: &graphene::Matrix) -> Option<Transform> {
+    pub fn matrix(&self, matrix: &graphene::Matrix) -> Transform {
```
Comment (sdroege) [inline gsk4/src/auto/transform.rs:52]: This is correct and can't return NULL

**Comment by sdroege** [comment]
@bilelmoussaoui Would you want to backport this or not?

**Comment by bilelmoussaoui** [comment]
> @bilelmoussaoui Would you want to backport this or not?

sure that sounds good to me

**Pull request merged**